sklearn/manifold/_isomap.py

```diff
@@ -169,7 +169,7 @@ class Isomap(ClassNamePrefixFeaturesOutMixin, TransformerMixin, BaseEstimator):
         "n_neighbors": [Interval(Integral, 1, None, closed="left"), None],
         "radius": [Interval(Real, 0, None, closed="both"), None],
         "n_components": [Interval(Integral, 1, None, closed="left")],
-        "eigen_solver": [StrOptions({"auto", "arpack", "dense"})],
+        "eigen_solver": [StrOptions({"auto", "arpack", "dense", "randomized_value"})],
```

Comment:
Please complete the docstring to add this new value and explain what happens.

Also, please add the same additional parameter as in kernelPCA
`iterated_power` (copy the docstring from kPCA)